docs/getting-started/new_project_guide.md

```diff
@@ -400,3 +400,19 @@ Use the [ClusterFuzz web interface](https://oss-fuzz.com/) to review the followi
 * Fuzzer performance analyzer (linked from fuzzer statistics)
 
 **Note:** Your Google Account must be listed in [project.yaml](#projectyaml) for you to have access to the ClusterFuzz web interface.
+
+### Status Badge
+
+![Example
+Badge](https://oss-fuzz-build-logs.storage.googleapis.com/badges/curl.svg)
+
+Once your project has started building, we'd love it if you added our badge in
```

Comment:
s/building/fuzzing ?

Reply:
I think the badge won't be created until the first build though?

Reply:
ah ok, can we hyperlink "building" to our build page - (https://oss-fuzz-build-logs.storage.googleapis.com/index.html)

Reply:
Done